tests/quotes/jsfmt.spec.js

```diff
@@ -1,2 +1,2 @@
 run_spec(__dirname, ["flow"]);
-run_spec(__dirname, ["flow"], { singleQuote: true });
+run_spec(__dirname, ["flow"], { jsxSingleQuote: true });
```

Comment:
Why was this changed?

Reply:
ah, artifact from the old functionality. will revert